scripts/make_release.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e # Fail on error
+
+ACTION='\033[1;90m'
+FINISHED='\033[1;96m'
+READY='\033[1;92m'
+NOCOLOR='\033[0m'
+ERROR='\033[0;31m'
+
+echo -e "${ACTION}Checking environnement${NOCOLOR}"
+if [[ ! $1 =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]
```

Comment:
should we use full sem ver regex ? (overkill IMHO)
https://semver.org/ (bottom)
https://regex101.com/r/Ly7O1x/3/

note: CMake also allow a "tweak" field
https://cmake.org/cmake/help/latest/command/project.html#options
note: I've never use it, but sometine it is use when you just want to bump a dependency/compiler config but your source code is not touched, not sure it is releavant for cpu_features

Reply:
Let's keep it simple for now and improve if we feel it's needed.